airflow/contrib/hooks/datastore_hook.py

```diff
@@ -27,8 +30,9 @@ def __init__(self,
         super(DatastoreHook, self).__init__(scope, datastore_conn_id, delegate_to)
         # datasetId is the same as the project name
         self.dataset_id = self._extras_dejson().get('project')
+        self.connection = self._get_conn()
 
-    def get_conn(self):
+    def _get_conn(self):
```

Comment:
Can you leave this public? There are use cases where getting the connection would be nice.